docs/field-details.asciidoc

```diff
@@ -6193,21 +6163,6 @@ Distributed tracing makes it possible to analyze performance throughout a micros
 
 // ===============================================================
 
-| trace.id
```

Comment:
> [ ] We need to fix issues around the tracing fields. Tracing is a special field set in that it's marked as `root=true` just like Base fields (meaning they're not nested under `tracing.`). However it contains two nested ID fields, `trace.id` and `transaction.id`. This rewrite didn't account for this, and one of them gets overwritten.

`trace.id` getting removed is related to the statement above?

Reply:
Yes this was the last major problem to solve. It was fixed by 98745caf5